<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientSettingsDialog</class>
 <widget class="QDialog" name="ClientSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tabNetwork">
        <attribute name="title">
         <string>Connection</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="serverIPLabel">
             <property name="text">
              <string>Server IP:</string>
             </property>
             <property name="buddy">
              <cstring>serverIPLineEdit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="serverIPLineEdit">
             <property name="inputMask">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="serverPortLabel">
             <property name="text">
              <string>Port:</string>
             </property>
             <property name="buddy">
              <cstring>serverPortSpinBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="serverPortSpinBox">
             <property name="maximum">
              <number>65535</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="cacheFilesBox">
             <property name="text">
              <string>Use cache for files recieved from server</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearCacheButton">
             <property name="text">
              <string>Clear cache...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="autoConnectBox">
           <property name="text">
            <string>Auto connect to server at start and load driver</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="driversListCombo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="loadDriversButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>43</width>
             <height>68</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabProtocol">
        <attribute name="title">
         <string>Protocol</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="writeProtocolCheckBox">
           <property name="text">
            <string>Write equipment protocol</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="protocolPathLabel">
             <property name="text">
              <string>Path:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="protocolPathEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="protocolPathSelectButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="protocolTimerLabel">
             <property name="text">
              <string>Save data to file every:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="protocolTimerSpinBox">
             <property name="suffix">
              <string> sec.</string>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="deleteProtocolAfterLabel">
             <property name="text">
              <string>Delete protocol files after:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="deleteProtocolFilesAfterSpinBox">
             <property name="suffix">
              <string> day(s)</string>
             </property>
             <property name="maximum">
              <number>365</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="protocolSaveToLabel">
             <property name="text">
              <string>Save to:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="protocolSaveToPathEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="protocolSaveToBrowseButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>125</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabInterface">
        <attribute name="title">
         <string>Interface</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QCheckBox" name="startInFullScreenCheckBox">
           <property name="text">
            <string>Start in full screen mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="saveWindowsPosCheckBox">
           <property name="text">
            <string>Save windows position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="restoreWindowsPosCheckBox">
           <property name="text">
            <string>Restore windows position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="showGridCheckBox">
           <property name="text">
            <string>Show grid</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="toolBarGroupBox">
           <property name="title">
            <string>Toolbar</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="toolBarButtonSizeLabel">
                <property name="text">
                 <string>Toolbar button size:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="toolBarButtonSizeComboBox">
                <item>
                 <property name="text">
                  <string>Lagre</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Medium</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Small</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Extra small</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>128</width>
                  <height>23</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QCheckBox" name="showToolBarButtonsTextCheckBox">
                <property name="text">
                 <string>Show toolbar buttons text</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>138</width>
                  <height>17</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="controlWidgetGroupBox">
           <property name="title">
            <string>Control panel</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QCheckBox" name="showDebugPageCheckBox">
              <property name="text">
               <string>Show debug page</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="showLogMessagesPageCheckBox">
              <property name="text">
               <string>Show log messages page</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabLogging">
        <attribute name="title">
         <string>Logging</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QCheckBox" name="clearLogsCheckBox">
           <property name="text">
            <string>Clear log files at start</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>293</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>serverIPLineEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ClientSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
